lib/buffer.js

```diff
@@ -1466,7 +1467,7 @@ Buffer.prototype.swap32 = function swap32() {
   // dropping down to the native code is faster.
   const len = this.length;
   if (len % 4 !== 0)
-    throw new RangeError('Buffer size must be a multiple of 32-bits');
+    throw new errors.RangeError('ERR_INVALID_BUFFER_SIZE', '32-bits');
```

Comment:
Since this code is for a little bit more specific condition, think about `ERR_INVALID_BUFFER_ALIGNMENT`, but this is just a suggestion. `ERR_INVALID_BUFFER_SIZE` might be useful for more situations in the future, so it's Ok too.
@starkwang it's your PR, so you should decide 👍